filament/include/filament/Skybox.h

```diff
@@ -64,6 +64,12 @@ class UTILS_PUBLIC Skybox : public FilamentAPI {
     struct BuilderDetails;
 
 public:
+    enum SkyboxType {
```

Comment:
nit: Filament seems to make use of scoped enums, so I suggest keeping that convention

Reply:
Fixed in https://github.com/shapr3d/filament/pull/8/commits/91d5e221aec0759129b6da5ad786383348ffd4a2